types/app/log.go

```diff
@@ -14,7 +14,7 @@ type LogWatcher interface {
 
 type AppLogService interface {
 	Enqueue(entry *Applog) error
-	Add(appName, message, source, unit string) error
+	Add(appName, message, source, unit string, level int) error
```

Comment:
I don't think it's a good idea to change this method signature. This method is only a helper used for log messages produced by tsuru itself. We can assume they will always have the same level and leave this method as it was before, reducing the diff for this PR.